data/campaigns/The_Rise_Of_Wesnoth/units/Wesfolk_Leader.cfg

```diff
@@ -19,7 +19,7 @@
     {AMLA_DEFAULT}
     cost=60
     usage=mixed fighter
-    description= _ "Born in the Wesfolk aristocracy, she and her people were excluded from society by their lords, who betrayed their loyalty when a war against Haldric’s people was being lost. This outcast still conserves her nobleness in her veins and on the battlefield she has earned valuable experience, which continues increasing with time, as does her natural leadership."
+    description= _ "Born in the Wesfolk aristocracy, she and her people were excluded from society by their lords, who betrayed their loyalty when a war against Haldric’s people was being lost. This outcast still conserves her nobleness in her veins, and on the battlefield she has earned valuable experience, which continues to increase with time, as does her natural leadership."
```

Comment:
This looks good, but I notice an awkward passive in here too - "was being lost". Is there anything we can do about that? It's not the end of the world, but it sure is awkward.

Reply:
Settled on "when they started losing a war against Haldric's people"